wurst/event/DamageType.wurst

```diff
@@ -155,12 +156,8 @@ function handler() returns bool
 
 init
     onEnter() ->
-        GetTriggerUnit()..addAbility(DAMAGE_TYPE_CHECK_ID)..makeAbilityPermanent(DAMAGE_TYPE_CHECK_ID, true)
-
-    forUnitsInRect(bj_mapInitialPlayableArea) iter ->
-        iter..addAbility(          DAMAGE_TYPE_CHECK_ID)
-            ..makeAbilityPermanent(DAMAGE_TYPE_CHECK_ID, true)
-
+        getEnterLeaveUnit()..addAbility(DAMAGE_TYPE_CHECK_ID)..makeAbilityPermanent(DAMAGE_TYPE_CHECK_ID, true)
```

Comment:
Why did you change this?

Reply:
onEnter does not operate with getTriggerUnit
It might work, thought it isn't guaranteed to work, especially if onunitenterleave will ever change somehow.